run_sir.py

```diff
@@ -14,11 +14,14 @@
 		print('usage: python3 run_sir.py [nwk file]')
 		exit()
 
+	## Print header
+	print('beta','outbreaksize','outbreaksizeerr','extinctiontime','extinctiontime')
```

Comment:
Sorry, left a typo in the header... Last was supposed to be `extinctiontimeerr` 👎